test/e2e/create_test.go

```diff
@@ -315,7 +315,7 @@ var _ = Describe("Podman create", func() {
 	It("podman create --authfile with nonexistent authfile", func() {
 		session := podmanTest.Podman([]string{"create", "--authfile", "/tmp/nonexistent", "--name=foo", ALPINE})
 		session.WaitWithDefaultTimeout()
-		Expect(session).To(Not(Equal(0)))
+		Expect(session).ToNot(HaveValue(Equal(0)))
```

Comment:
This does not look right. 

I assume the test wants to check the exit code:
`Expect(session).To(Exit(125))`
The test is very broken for a long time: https://github.com/containers/podman/commit/d7c0f968ca60994306c8c76cd8e4e0a677fe9ada and thus failed to catch a regression,
compare https://github.com/containers/podman/pull/18413, for consistency this should error out as well.

Reply:
Not only the test is broken but also the code.  It should error but doesn't. I'll change the test but will also open an issue.

Reply:
https://github.com/containers/podman/issues/18938

Reply:
yes that is what I meant with it failed to catch this regression.